_updates/2020-02-12-february.md

```diff
@@ -0,0 +1,39 @@
+---
+title: February 2020
+---
+
+## 💛💙Valentine's Day 2020💜💚
+
+It is the time to love and be loved. 
+Here are the changes what we made last month. We hope that you will love them 💗👇
+In this release we were about to optimize the admin-ui.
+
+## Admin-ui@1.4.3
+
+We fixed the 'add new content' dialog [#538](https://github.com/SenseNet/sn-client/pull/538)
+
+We fixed some bugs on the drawer [#526](https://github.com/SenseNet/sn-client/pull/526)
+
+
+We started using relative-ci-agent webpack plugin
+This way we don't have to:
+- Run a new travis job
+- Less configuration and code in snapp 
+[#545](https://github.com/SenseNet/sn-client/pull/545)
+
+We had to move the Search button from bottom-right because it it a dedicated place for out chat application. Now you can use clicking the add button for search in Search menu.
```

Comment:
```suggestion
We had to move the Search button from bottom-right because it is a dedicated place for out chat application. Now you can use clicking the add button for search in Search menu.
```